CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Automatically generated group names are now converted from LaTeX to Unicode
 - Unified dialogs for opening/saving files
 - Add integrity check to avoid non-ASCII characters in BibTeX files
+- Updated German translation
```

Comment:
I do not see any need for this. Or rather, no other languages state the updates. (I think French, Turkish, and Swedish are updated recently as well.) Maybe we should collect this in a single point before each release?